### ISSUE TITLE
Adding the animated emojis to this repo?

### ISSUE THREAD

**Issue by Haxorzz** [microsoft/fluentui-emoji#51]
as mentioned in this article: https://www.linkedin.com/pulse/making-fluent-emoji-nando-costa/ A lot of these emojis have been animated and I think since it's part of "fluent emojis" they should be in this repository?

**Comment by Reevak05** [comment]
Voicing support for this. I think the animation is what makes the Fluent Design emoji really special and appealing. It would be great to have them in this repo.

Also, thank you @jasoncuster and @spencer-nelson for your work on this and making it open source!

**Comment by DrinkCupboard** [comment]
Adding the animated emoji would make this set otherworldly! No pressure though, it's an amazing resource as-is. 👍

**Comment by jasoncuster** [comment]
That's definitely something we're talking about. 💖

**Comment by bhyoo99** [comment]
❤️

**Comment by leonschwanitz** [comment]
@jasoncuster Is it allowed to use the animated emojis in commercial products? They can be downloaded at https://emojipedia.org/, when not available here.

**Comment by koffietiger** [comment]
@leonschwanitz @svszot - Think we can safely assume, not for the moment, as the licensing that allows us to use their emoji is directly tied to this repo. 😊

**Comment by spencer-nelson** [comment]
While files can be downloaded/scraped from sites like Emojipedia there is no current explicit open licensing for any of the Fluent Emoji outside of this repository. As noted previously, it's something we are discussing/exploring but haven't yet been able to conclude said discussion.

**Comment by aMySour** [comment]
> @jasoncuster That's definitely something we're talking about. sparkling_heart

Any update? It's been months now with nothing

**Comment by Not-Qualified** [comment]
@jasoncuster and @spencer-nelson ,

What happened to making animated emojis available discussion of yours ?

Appreciated all Your Help with Open Sourcing Emojis with 3D, Color, Flat and Glyph types

Can you provide information about more on animated emojis ?

**Comment by raffi-46** [comment]
Hey @jasoncuster and @spencer-nelson,

Could you please provide us with information about the introduction of animated emojis? We eagerly await your response, rather than turning to https://emojipedia.org/ for answers.

Please reply @jasoncuster and @spencer-nelson
 
We greatly appreciate your efforts.

**Comment by Not-Qualified** [comment]
@jasoncuster and @spencer-nelson ,

Would you be able to give a roughly idea that whether Microsoft is going to make Animated GIF Open Source or not ?

**Comment by arjunj132** [comment]
Is this still a thing? I love the animate emoji, but don't know if I can use them for my projects.

**Comment by Not-Qualified** [comment]
> Is this still a thing? I love the animated emoji, but don't know if I can use them for my projects.

Yes, It's still a thing and we are not clear whether we can use it freely or not

**Comment by huw** [comment]
Please don’t add comments in support of this or needlessly harass the repo owners—a number of people are subscribed to this thread and get a useless notification every time you do. You are not going to make this process any faster. It is enough to add a 👍 to the original issue.

**Comment by MichaelPriebe** [comment]
If anyone wants the emojis straight from the source, here is the manifest:

https://statics.teams.cdn.live.net/evergreen-assets/personal-expressions/v1/metadata/8c8abd7cb57944368fb1f1edf2274059/en-us.json

The emojis themselves:
https://statics.teams.cdn.office.net/evergreen-assets/personal-expressions/v2/assets/emoticons/cool/default/30_f.png
![](https://statics.teams.cdn.office.net/evergreen-assets/personal-expressions/v2/assets/emoticons/cool/default/30_f.png)

https://statics.teams.cdn.office.net/evergreen-assets/personal-expressions/v2/assets/emoticons/cool/default/100_f.png
![](https://statics.teams.cdn.office.net/evergreen-assets/personal-expressions/v2/assets/emoticons/cool/default/100_f.png)

The animation sprite sheets: (very tall!)
https://statics.teams.cdn.office.net/evergreen-assets/personal-expressions/v2/assets/emoticons/cool/default/30_anim_f.png

https://statics.teams.cdn.office.net/evergreen-assets/personal-expressions/v2/assets/emoticons/cool/default/100_anim_f.png

They seem to only exist in these sizes, so Emojipedia either found a different source, or was given to them by Microsoft. (shame!)

If anyone finds higher quality sources let me know.
